### PR TITLE
CORE-9189: added a table to keep track of iRODS tickets that have bee…

### PR DESCRIPTION
**Pull request by slr71**
…n generated for jobs

### CODE REVIEW

File: src/main/tables/82_job_tickets.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
+SET search_path = public, pg_catalog;
+
+--
+-- Records iRODS tickets that were generated for DE jobs.
+--
+CREATE TABLE job_tickets(
+   job_id uuid NOT NULL,
+   ticket varchar(16) NOT NULL,
```
Comment (ianmcorvidae) [inline src/main/tables/82_job_tickets.sql]: This looks like too short a field to me -- the tickets we generate are usually UUIDs

**Review by ianmcorvidae — COMMENTED** [review]
One relatively small thing, but in general looks good.

File: src/main/tables/82_job_tickets.sql (hunk 1 above)
Comment (slr71) [reply src/main/tables/82_job_tickets.sql]: Oh, okay. I based the field length on the tickets generated by `iticket`. Thanks for catching that.

Comment (ianmcorvidae) [reply src/main/tables/82_job_tickets.sql]: It looks like the ICAT itself uses a `varchar(100)`, if you just want to copy that!

Comment (slr71) [reply src/main/tables/82_job_tickets.sql]: Will do.

**Comment by slr71** [comment]
Thanks for the review!

**Pull request merged**